keyboards/viterbi/rev2/rev2.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "viterbi.h"
+#include "quantum.h"
+
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+#define LAYOUT( \
```

Comment:
Honestly, I'd like to remove the `LAYOUT` and just use `LAYOUT_ortho_5x14` here. 

```suggestion
#define LAYOUT_ortho_5x14 ( \
```

But this is optional (and would require removing the define below, and updating `viterbi.h`